### PR TITLE
Add option to export lectures

### PR DESCRIPTION
**Pull request by Bozydarek**
This PR add option to export lectures in a format used by the scheduler-spreadsheet created by @Arsenicro.

### CODE REVIEW

File: app/src/users/views.py [hunk 1]
```diff
@@ -294,3 +295,19 @@ def list_csv_preferences_paid(request):
...
+
+@staff_member_required
+@require_http_methods(['GET'])
+def list_csv_lectures(request):
+    lectures = Lecture.objects.all()
+    header = ("Name", "Printed name", "Lecturers", "Duration", "Type",
+              "IsSponsor", "Comment")
+    data = [(
+        str(lecture.title), "", str(lecture.author), str(lecture.duration),
+        str([lt[1] for lt in LECTURE_TYPE if lt[0] == lecture.lecture_type][0]),
```
Comment (Bozydarek) [inline app/src/users/views.py:308]: @ref-humbold is there any built-in mechanizm for this?

Comment (ref-humbold) [reply app/src/users/views.py:308]: I'd go for `dict(LECTURE_TYPE)[lecture.lecture_type]`.

File: app/src/users/views.py [hunk 2]
```diff
@@ -294,3 +295,19 @@ def list_csv_preferences_paid(request):
         ) for p in prefs
     ]
     return csv_response(header, data_list, filename="list_csv_preferences_paid")
+
+
+@staff_member_required
+@require_http_methods(['GET'])
+def list_csv_lectures(request):
+    lectures = Lecture.objects.all()
```
[the remark this answers is not in the document]
Comment (Arsenicro) [reply app/src/users/views.py:303]: To be honest, both are fine. We don't want to schedule unaccepted lectures, but we can not put them into the plan.

[the remark this answers is not in the document]
Comment (Bozydarek) [reply app/src/users/views.py:303]: It was my first thought too, but as I understand this scheduler-spreadsheet is supposed to help create a schedule, so it should have all lectures. @Arsenicro, what do you think about it?

[the remark this answers is not in the document]
Comment (Arsenicro) [reply app/src/users/views.py:303]: Let's leave it like this now; as I said, both are fine. However, we may want to check if we can fit the lecture before accepting it, so maybe this is more practical.

File: app/src/users/views.py [hunk 3]
```diff
@@ -294,3 +295,19 @@ def list_csv_preferences_paid(request):
...
+    header = ("Name", "Printed name", "Lecturers", "Duration", "Type",
```
[the remark this answers is not in the document]
Comment (Bozydarek) [reply app/src/users/views.py:304]: Not at the moment, but I can easily implement it. However I don’t think that it is the best approach. Having a header for your data can help you verify if they are correct and removing a line from a file or a spreadsheet when it is not needed is easy :wink:

**Pull request merged**